Sources/ShopifyAcceleratedCheckouts/ShopifyAcceleratedCheckouts+Errors.swift

```diff
@@ -0,0 +1,52 @@
+/*
+ MIT License
+
+ Copyright 2023 - Present, Shopify Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import Foundation
+
+extension ShopifyAcceleratedCheckouts {
+    enum InvariantMessages: String {
+        case nilCart = "cart is nil."
+        case nilDisplayName = "displayName is nil."
+        case nilPayment = "payment is nil."
+        case nilPaymentData = "paymentData is nil."
+        case nilEmail = "email is nil."
+        case nilShippingMethodID = "shippingMethodID is nil."
+        case nilShippingMethod = "shippingMethod is nil."
+        case nilBillingAddress = "billingAddress is nil."
+        case nilLastDigits = "lastDigits is nil."
+        case nilPostalAddress = "postalAddress is nil."
+        case nilBillingContact = "billingContact is nil."
+        case nilShippingContact = "shippingContact is nil."
```

Comment:
Could this just be a dynamic string?

Reply:
Sounds good 👍

Reply:
Yes, they could. What would you propose? Something like `{field} is nil.`

Having them as a list has its pros, like consistency, we have in one place the messages that will be used (easy to review/update). But it's true that is verbose.

In any way, these values are not being used yet, so if we refactor it now here it would be "half" done. If we decide to refactor I suggest doing it in the next PR to tackle it all at once and make sure it works fine.

Reply:
Yeah `{field} is nil.` would suffice and be consistent. The type can probably just be a `String` unless for some reason we need to know specifically which field is nil